include/triton/core/tritonbackend.h

```diff
@@ -561,6 +561,16 @@ TRITONBACKEND_RequestOutputBufferProperties(
 TRITONBACKEND_DECLSPEC TRITONSERVER_Error* TRITONBACKEND_RequestRelease(
     TRITONBACKEND_Request* request, uint32_t release_flags);
 
+/// Get the trace associated with a request. The returned trace is owned by the
+/// request, not the caller, and so should not be modified or freed.
+/// If tracing is disabled, then `nullptr` will be returned.
```

Comment:
```suggestion
/// If the request is not being traced, then `nullptr` will be returned.
```
Tracing can be configured to sample 1 for every few requests.